less/common.less

```diff
@@ -113,6 +122,11 @@ html.no-js .no-js-warning {
   border-top: 0.2rem solid @white;
 }
 
+#content {
+  padding-left: 0;
+  padding-right: 0;
+}
```

Comment:
Oh, can you set negative margin to offset the padding? I actually did that with the hero unit for the original site, way back in February: https://github.com/mozilla/teach.webmaker.org/blob/8804e410ce96df5d2983fe829b6a81c335e94d74/styles.less#L23-L26

I'm not sure if that CSS has actually survived to present-day or not, hehe.